src/difference.js

```diff
@@ -0,0 +1,19 @@
+function difference(a, b) {
+  var hash = {}
+  var diff = {}
+
+  b.forEach(function createHash(item) {
+    hash[item] = true
+  })
+
+  a.forEach(function findDiff(item) {
+    if (!hash[item] && !diff[item]) {
+      diff[item] = true
+    }
+  })
+
+  return Object.keys(diff)
```

Comment:
I don't remember that `Object.keys` or `Object.prototype.forEach` are part of v8's language implementations in node `0.10` (or `0.12`). It could possibly just work within the tests' environment because they're transpiled with babel...

`Object.property.forEach` even isn't spec, or is it?